tests/integration/piv_test.py

```diff
@@ -1298,6 +1317,11 @@ def test_piv_full_back_ok_on_stops(mock_rabbitmq):
 
 
 def test_piv_added_stop_time_at_the_beginning(mock_rabbitmq):
+    """
+    schedule is added on stop 0 departure
+    stop is added after the stop 0
+    schedule is added on stop 2 (previously 1)  arrival
```

Comment:
Not sure about "rang":
```suggestion
    new stop 0 is added with only departure
    new stop 1 is added after the stop 0
    arrival is added on stop 2 (previously 0)
```